src/Squirrel/ReleaseEntry.cs

```diff
@@ -259,22 +259,17 @@ public static List<ReleaseEntry> BuildReleasesFile(string releasePackagesDir)
             var entries = entriesQueue.ToList();
             var tempFile = default(string);
 
-            // PO: needs fix, does not create any file (redmine #16652)
-            Utility.WithTempFile(out tempFile, releasePackagesDir);
-
-            try {
+            using (Utility.WithTempFile(out tempFile, releasePackagesDir)) {
                 using (var of = File.OpenWrite(tempFile)) {
                     if (entries.Count > 0) WriteReleaseFile(entries, of);
-                }
+                 }
```

Comment:
Why to add those spaces?

Reply:
It was added unintentionally. I just removed them.